### PR TITLE
Added some fields for migration

### PR DESCRIPTION
**Pull request by AssossCare**

### CODE REVIEW

File: src/main/java/org/taktik/icure/entities/embed/Employer.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package org.taktik.icure.entities.embed;
+
+import java.util.Objects;
+
+public class Employer {
```
Comment (antoinepairet) [inline src/main/java/org/taktik/icure/entities/embed/Employer.java]: Should be a `Person`

File: src/main/java/org/taktik/icure/entities/embed/Employer.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    private String name;
+    private Address addresse;
```
Comment (antoinepairet) [inline src/main/java/org/taktik/icure/entities/embed/Employer.java]: English should be used

File: src/main/java/org/taktik/icure/services/external/rest/v1/dto/PatientDto.java [hunk 3]
```diff
@@ -70,6 +74,7 @@
 	protected String warning;
     protected String nationality;
 	protected String preferredUserId;
+	protected String comment;
```
Comment (antoinepairet) [inline src/main/java/org/taktik/icure/services/external/rest/v1/dto/PatientDto.java]: There is already an administrative note.
What is the difference?

Comment (AssossCare) [reply src/main/java/org/taktik/icure/services/external/rest/v1/dto/PatientDto.java]: administrative note is use for post-it and post-it is different from comment

**Pull request merged**